CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing to Auth0 projects
+
+A big welcome and thank you for considering contributing to the Auth0 open source projects. It’s people like you that make it a reality for users in our community.
```

Comment:
Looks like we are opening this repo for contributions. Do we have a plan on how to use team's bandwidth on this commitment.

Reply:
I updated the contributing file to redirect them into opening a ticket. I removed "submitting a pull request" section